src/core/thread/thread_netif.hpp

```diff
@@ -197,15 +198,16 @@ class ThreadNetif : public Ip6::Netif
 #if OPENTHREAD_CONFIG_SNTP_CLIENT_ENABLE
     Sntp::Client mSntpClient;
 #endif
-    MeshCoP::ActiveDataset  mActiveDataset;
-    MeshCoP::PendingDataset mPendingDataset;
-    Ip6::Filter             mIp6Filter;
-    KeyManager              mKeyManager;
-    Lowpan::Lowpan          mLowpan;
-    Mac::Mac                mMac;
-    MeshForwarder           mMeshForwarder;
-    Mle::MleRouter          mMleRouter;
-    Mle::DiscoverScanner    mDiscoverScanner;
+    MeshCoP::ActiveDataset        mActiveDataset;
+    MeshCoP::PendingDataset       mPendingDataset;
+    MeshCoP::ExtendedPanIdManager mExtendedPanId;
```

Comment:
nit:
```suggestion
    MeshCoP::ExtendedPanIdManager mExtendedPanIdManager;
```

May be confusing as there is another `ExtendedPanId` class.